internal/bootstrapping/agent.go

```diff
@@ -407,11 +402,11 @@ func (a *Agent) handleResponseMessage(msg *message.Message) (string, *MessageErr
 					bsRequestID:           response.ID,
 				})
 
-				return replyToID, msgError, err
+				return correlationID, respReq, msgError, err
 			}
 		}
 	}
-	return replyToID, nil, nil
+	return correlationID, respReq, nil, nil
```

Comment:
Isn't it an error situation? Basically, this means that the bootstrapping receives a message which is not addressed to the correct device or path is wrong. In this case, the message should be ignored. Currently, we can send a response.